CHANGELOG.md

```diff
@@ -5,6 +5,37 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v10.0.0](https://github.com/puppetlabs/puppetlabs-postgresql/tree/v10.0.0) - 2023-08-29
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-postgresql/compare/v9.1.0...v10.0.0)
+
+### Added
+
+- Add Debian 12 compatibility [#1460](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1460) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
I think Debian 12 should be added as a supported system to metadata and included in acceptance tests first. See https://github.com/puppetlabs/litmusimage/pull/62